ci/setup_env.sh

```diff
@@ -10,9 +10,9 @@ export COMMON_FILE_LOCATION='ci/common.sh'
 workdir="$( cd "$( dirname "${BASH_SOURCE[0]}" )/.." &> /dev/null && pwd )"
 github_tag="${GITHUB_REF_NAME:-}"
 release_branch="${RELEASE_BRANCH:-release}"
-prod_release_regex='^[0-9]+\.[0-9]+\.[0-9]+$'
-dev_release_regex='^[0-9]+\.[0-9]+\.[0-9]+(-rc[0-9]+){1}$'
-test_release_regex='^[0-9]+\.[0-9]+\.[0-9]+-[a-zA-Z0-9]+$'
+prod_release_regex='^[0-9]+\.[0-9]+\.[0-9]+\-[0-9]+\.[0-9]+\.[0-9]+$'
```

Comment:
Again, same here this regex will enforce 1.1.1-2.2.2 format and will not match 1.1.1 for example

Reply:
Yes that's correct, on the GitHub tags side there will be nomore `X.Y.Z` but only `X.Y.Z-A.B.C`; slighlty different for Docker tags.